src/pages/Aggregator/Aggregator.utils.ts

```diff
@@ -33,3 +43,67 @@ export const removeFocusFromCurrentField = ({
     node.style.color = '#fff';
   }
 };
+
+export function useIsFormValid(state: {
+  startingToken: string;
+  destinationToken: string;
+  startingChain: ChainEnum;
+  destinationChain: ChainEnum;
+  amount: string;
+  receivingAddress: string;
+  isAddressDisclaimerChecked: boolean;
+  isStartingTokenPaused: boolean;
+}): boolean {
+  const startingTokenBalance = useSelector(startingTokenBalanceSelector) ?? 0;
+  const destinationTokenAggregatorBalance = useSelector(
+    destinationTokenAggregatorBalanceSelector
+  ) ?? 0;
+  const startingTokenDecimals = useSelector(startingTokenDecimalsSelector);
+
+  // validate the tokens
+  if (
+    !(isRSK(state.startingChain) || isRSK(state.destinationChain)) ||
```

Comment:
This is definitely not correct - you would basically validate every conversion except `RSK -> RSK` as invalid.